examples/cpp/CMakeLists.txt

```diff
@@ -1,8 +1,15 @@
 cmake_minimum_required(VERSION 3.20)
 
-set(OPENDAQ_EXAMPLES_VERSION ${OPENDAQ_PACKAGE_VERSION})
+if(NOT DEFINED OPENDAQ_PACKAGE_VERSION)
+  set(OPENDAQ_EXAMPLES_VERSION "0.0.0")
+elseif ()
+  set(OPENDAQ_EXAMPLES_VERSION ${OPENDAQ_PACKAGE_VERSION})
+endif()
+
 project(opendaq_examples VERSION ${OPENDAQ_EXAMPLES_VERSION})
 
+add_compile_definitions(MODULE_PATH="${OPENDAQ_MODULES_DIR}")
```

Comment:
I would call this `OPENDAQ_MODULE_PATH` (and change the code that uses this variable).

But also: shouldn't this be `target_compile_definition`?